### PR TITLE
Ensure we are able to checkout with cybersource and load the paypal payment page

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
The routing code in ecommerce was assuming a certain structure. The structure was different here so I was making changes to fix that.
I also changed what we are sending to cybersource to fix some errors I ran into when trying to check out.
Paypal checkout still didn't work for me but I'm not sure if it works locally, was hoping to test on stage or eventually prod

### CODE REVIEW

File: public/static-checkout/js/scripts.js [hunk 1]
```diff
@@ -3,84 +3,87 @@
...
-    UT: 'Utah',
-    VT: 'Vermont',
-    VA: 'Virginia',
-    WA: 'Washington',
-    WV: 'West Virginia',
-    WI: 'Wisconsin',
-    WY: 'Wyoming',
-  },
+  CA: [
+    ['AB', 'Alberta'],
```
Comment (MatthewPiatetsky) [inline public/static-checkout/js/scripts.js]: need to send the short codes to cybersource on checkout

File: public/static-checkout/js/scripts.js [hunk 2]
```diff
@@ -215,9 +219,9 @@ paypalButton.addEventListener('click', function submitPaypal() {
       }
     }
   };
-  var basketId = getBasketId();
+  var basket_id = getBasketId();
   xhr.send(JSON.stringify({
-    basketId,
+    basket_id,
```
Comment (MatthewPiatetsky) [inline public/static-checkout/js/scripts.js]: ecommerce expects basket id in snake case

File: webpack.prod.config.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ config.plugins = [
   // Stage also uses this config and dev uses this config when running npm run build
   new CopyPlugin([
     {
-      from: path.resolve(__dirname, 'public/REV1074'),
+      from: path.resolve(__dirname, 'public/static-checkout'),
```
Comment (MatthewPiatetsky) [inline webpack.prod.config.js]: this is what pulls in the file into the dist folder (necessary to access it on prod)

File: public/static-checkout/js/scripts.js [hunk 4]
```diff
@@ -108,8 +111,8 @@ window.onclick = function closeMenu(event) {
 };
 
 function redirectToMFE(couponCode) {
-  var url = window.location.href;
-  var sku = /static-checkout\/([^&#]*).html/.exec(url)[1];
+  var url = window.location.pathname;
+  var sku = /.*\/([^&#]*).html/.exec(url)[1];
```
Comment (MatthewPiatetsky) [inline public/static-checkout/js/scripts.js]: this regex wasn't working for me

File: public/static-checkout/js/scripts.js [hunk 5]
```diff
@@ -118,8 +121,8 @@ function redirectToMFE(couponCode) {
 }
 
 function getBasketId() {
-  var url = window.location.href;
-  var basketParams = /[?&]basketId=([^&#]*)/.exec(url);
+  var url = window.location.search;
+  var basketParams = /[?&]basket_id=([^&#]*)/.exec(url);
```
Comment (MatthewPiatetsky) [inline public/static-checkout/js/scripts.js]: this regex wasn't working for me

File: public/static-checkout/js/scripts.js (hunk 4 above)
Comment (julianajlk) [reply public/static-checkout/js/scripts.js]: Why wasn't it working? I'm curious because I get the sku when I try on console

Comment (MatthewPiatetsky) [reply public/static-checkout/js/scripts.js]: not sure, i get that now too but pretty sure at one point it didn't work

**Pull request merged**